static_frame/core/index.py

```diff
@@ -430,6 +430,13 @@ def __init__(self,
         # NOTE: automatic discovery is possible, but not yet implemented
         self._loc_is_iloc = loc_is_iloc
 
+    def __reversed__(self):
```

Comment:
I'm gradually trying to get the codebase to pass static type checking, so a type hint here would be great:

```suggestion
    def __reversed__(self) -> tp.Iterator[tp.Any]:
```

(`Index` isn't `Generic` yet, so `tp.Any` as the item type is fine for now.)

Reply:
Done.